frontend/src/pages/ConnectedPages/Game/UsersInGame/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { UserInGame } from "../../../../websockets/MessageType";
+
+import "./usersingame.css";
+
+interface IUsersInGame {
+  users: UserInGame[],
+}
+
+const maxUsersToDisplay = 14;
+const maxCharactersPerName = 8;
+
+const UsersInGame: React.FC<IUsersInGame> = ({ users: users }: IUsersInGame): JSX.Element => {
+  const totalPlayers = users.length;
+  const totalColumns = 3;
+
+  const setRowClassName = (index: number): string => {
+    return (index + 1) % 2 ? "usersInGameEvenRowColor" : "usersInGameParRowColor";
+  }
+
+  const setColumnClassName = (row: number, col: number): string => {
+    let className = "usersInGameColumn";
+
+    if (row === 0 && col === 0) {
+      if (totalPlayers === 1) {
+        return className += " usersInGameFullLeftCorner"
+      }
+      return className + " usersInGameTopLeftCorner"
+    }
+    if (row === totalPlayers - 1 && col === 0) {
+      return className + " usersInGameBottomLeftCorner"
+    }
+    if (row === 0 && col === totalColumns - 1) {
+      if (totalPlayers === 1) {
+        return className += " usersInGameFullRightCorner"
+      }
+      return className + " usersInGameTopRightCorner"
+    }
+    if (row === totalPlayers - 1 && col === totalColumns - 1) {
+      return className + " usersInGameBottomRightCorner"
+    }
+
+    return className;
+  }
```

Comment:
Could you add above this a TODO for me to change this with css ? Like my comment with the even/par, I just saw that we can do this with the `:nth-of-type` and `:nth-last-of-type`